x-pack/plugin/sql/sql-action/src/test/java/org/elasticsearch/xpack/sql/action/SqlQueryRequestTests.java

```diff
@@ -242,6 +259,9 @@ private static void toXContent(SqlQueryRequest request, XContentBuilder builder)
         if (request.cursor() != null) {
             builder.field(CURSOR_NAME, request.cursor());
         }
+        if (request.runtimeMappings() != null && request.runtimeMappings().isEmpty() == false) {
```

Comment:
Same here, why do we need to check if it's empty?
maybe an assertion that are empty could be better?